app/grandchallenge/evaluation/views/__init__.py

```diff
@@ -280,6 +280,19 @@ class MethodDetail(
     login_url = reverse_lazy("account_login")
 
 
+class MethodImportStatusDetail(
+    LoginRequiredMixin,
+    ObjectPermissionRequiredMixin,
+    CachedPhaseMixin,
```

Comment:
Why is `CachedPhaseMixin` needed?

Reply:
I think it's not needed; but I'm not sure. I saw that it was used for the MethodDetail view and assumed I needed it here as well. But when I look at the commit in which it was added, I don't understand why it is needed for the MethodDetail view at all (and I think it might not be needed there either.)

Reply:
Take a read at what the Mixin is doing - it is adding a context variable and form kwargs. You don't have a form in your view so there is no need for the latter, and I don't think that you're using the `phase` variable in your template, so, I do not think that it is necessary.

Reply:
Thanks for the explanation .I think it's not needed in the MethodDetail view either; but I left it there since it's not part of this PR.